### DISCUSSION TITLE
Alpine Linux, Neovim and Copilot works only for Bash

### DISCUSSION THREAD

**Discussion by mcharvat90** [community/community#61780] [Copilot Conversations]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Dear team,
I am facing issue on my Alpine distro 3.17 with installed and activated Github copilot as pluin for NeoVIM.
Basically suggestions works for Bash scripts but not for Python3 edited programs (it does not shows anything even in :Copilot subwindow exept that duplicate suggestions will be not displayed). The issue seems to be related to LSP client not being available, the log (:Copilot log) after few moments looks as follows:
`[INFO] [ghostText] [2023-07-25T23:00:32.025Z] Cancelled after extractPrompt
agent exited with status -1
Exception: copilot#agent: LSP client not available @ function <SNR>22_Trigger[6]..copilot#Suggest[2]..copilot#Complete[6]..copilot#Request[2]..<SNR>29_LspRequest, line 8
~ `

Installed versions:
Alpine: 1.37
NodeJS: 18.17.0
NeoVIM: 0.8.1
NPM: 9.1.2 

:checkhealth within nvim resulted in OK except ruby/gem (dont plan to use}

Probably I have missed something? Thanks a lot for the help,

Michal

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @mcharvat90 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
